manifests/templates/cicd/image-build.yaml

```diff
@@ -0,0 +1,39 @@
+- apiVersion: cyclone.io/v1alpha1
+  kind: Stage
+  metadata:
+    name: image-build-template
+    labels:
+      cyclone.io/stage-template: "true"
+      cyclone.io/scene: cicd
+      cyclone.io/template-kind: image-build
+    annotations:
+      cyclone.io/description: >
+        Build docker image with Dockerfile and build context
+  spec:
+    pod:
+      inputs:
+        arguments:
+        - name: image-name
+        - name: build-context
+          value: .
+        - name: dockerfile-path
+          value: ./Dockerfile
+        - name: image
+          value: docker:18.03.1-ce-dind
+        - name: cmd
+          value: |
+            cd /workspace/<repo-name> &&
+            docker build -f {{ dockerfile-path }} -t {{ image-name }} {{ build-context }}
+        resources:
+        - name: code
+          path: /workspace
+      outputs:
+        resources:
+        - name: image
```

Comment:
Can we give the `image` a default value `{{ image-name }}`?

Reply:
Image name is specified in the image resource definition, here is ref to the image resource